test_scripts/Smoke/API/050_SetMenuLayout_unspported_WARNINGS.lua

```diff
@@ -0,0 +1,79 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
typo in the name "unspported"

Reply:
fixed in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2213/commits/9ff4d86aca9eae938af767e253c57d09fa0fec37